src/script/script.h

```diff
@@ -212,26 +212,6 @@ enum opcodetype
     OP_IDENTITY_UPDATE = 0xd5,
     OP_IDENTITY_MULTISIG = 0xd6,
 
-    // distributed exchange
-    OP_OFFER_ACTIVATE = 0x04,
-    OP_OFFER_UPDATE = 0x05,
-    OP_OFFER_ACCEPT = 0x06,
-    OP_OFFER_ACCEPT_FEEDBACK = 0x07,
-
-    // distributed licensing system
```

Comment:
Is it worth leaving the OP_CERT_* in the codebase and remove the rest that won't be needed in the future for identity/certs.

Reply:
Identities is what would be immediately implemented without much change, although Certs would require a good amount of change so it wouldn't be in the immediate future, so I removed the cert OPCODES